src/proposals/20240417/SparkGnosis_20240417.t.sol

```diff
@@ -0,0 +1,306 @@
+            ltv:                      0,
+            liquidationThreshold:     0,
+            liquidationBonus:         0,
+            liquidationProtocolFee:   0,
+            reserveFactor:            10_00,
+            usageAsCollateralEnabled: false,
+            borrowingEnabled:         true,
+            interestRateStrategy:     _findReserveConfigBySymbol(allConfigsAfter, 'EURe').interestRateStrategy,
+            stableBorrowRateEnabled:  false,
+            isPaused:                 false,
+            isActive:                 true,
+            isFrozen:                 false,
+            isSiloed:                 true,
+            isBorrowableInIsolation:  false,
+            isFlashloanable:          true,
+            supplyCap:                5_000_000,
+            borrowCap:                4_000_000,
+            debtCeiling:              0,
+            eModeCategory:            0
+        });
+        _validateReserveConfig(eure, allConfigsAfter);
+        _validateInterestRateStrategy(
+            eure.interestRateStrategy,
+            eure.interestRateStrategy,
+            InterestStrategyValues({
+                addressesProvider:             address(poolAddressesProvider),
+                optimalUsageRatio:             0.9e27,
+                optimalStableToTotalDebtRatio: 0,
+                baseStableBorrowRate:          0.07e27,  // Equal to variableRateSlope1 as we don't use stable rates
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            0.07e27,
+                variableRateSlope2:            0.5e27
+            })
+        );
+        _validateAssetSourceOnOracle(poolAddressesProvider, EURE, EURE_PRICE_FEED);
+    }
+
+    function test_reserveChanges() public {
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+        
+        ReserveConfig memory daiConfigBefore    = _findReserveConfigBySymbol(allConfigsBefore, 'WXDAI');
+        ReserveConfig memory gnoConfigBefore    = _findReserveConfigBySymbol(allConfigsBefore, 'GNO');
+        ReserveConfig memory wstethConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'wstETH');
+        
+        IDefaultInterestRateStrategy oldInterestRateStrategy = IDefaultInterestRateStrategy(
+            daiConfigBefore.interestRateStrategy
+        );
+        assertEq(daiConfigBefore.ltv,           70_00);
+        assertEq(daiConfigBefore.supplyCap,     10_000_000);
+        assertEq(daiConfigBefore.borrowCap,     8_000_000);
+        assertEq(daiConfigBefore.reserveFactor, 0);
+        InterestStrategyValues memory interestStrategyValuesBefore = InterestStrategyValues({
+            addressesProvider:             address(poolAddressesProvider),
+            optimalUsageRatio:             oldInterestRateStrategy.OPTIMAL_USAGE_RATIO(),
```

Comment:
Can we use hard values for the `optimalUsageRatio` and `variableRateSlope2`? Just so I can verify against forum